src/utils.js

```diff
@@ -35,15 +35,17 @@ const utils = {
    * Fetches content from the given uri
    * @param {String} uri URL to fetch
    * @param {Logger} logger the logger
+   * @param {object} auth authentication object
```

Comment:
It would help if the structure of `auth` object would be documented, or at least provide a link to the documentation.

Reply:
ok, I'll add it.